libs/functions/generate_shapefiles.py

```diff
@@ -33,6 +33,7 @@ def join_and_output_shapefile(
         "State Intervention",
         "PEAK-HOSP",
         "PEAK-DEATHS",
+        "Hospital Over Capacity On",
```

Comment:
Not sure if this is the suggested field, but
```suggestion
        "Hospital Overcapacity Date",
```